sys-libs/libcpuid/libcpuid-0.4.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit autotools-utils
+
+DESCRIPTION='A small C library for x86 CPU detection and feature extraction'
+HOMEPAGE="http://${PN}.sourceforge.net"
+SRC_URI="https://github.com/anrieff/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE='BSD-2'
+SLOT='0'
+KEYWORDS='~amd64 ~x86'
+IUSE='test'
+RESTRICT='primaryuri'
```

Comment:
Why?

Reply:
Thank for your comment, but I do not understand your "why?".

Reply:
I mean, why do you put this RESTRICT rule in here. I think it's necessary only when upstream wants to track download count, and Github doesn't seem to do that. 

https://devmanual.gentoo.org/general-concepts/mirrors/index.html